mrp_lock_lot/__openerp__.py

```diff
@@ -34,18 +35,12 @@
         "Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>",
         ],
     "category": "Manufacturing",
-    "depends": ["stock",
-                "mrp",
-                "product",
+    "depends": ["mrp",
+                "stock_lock_lot",
                 ],
     "data": [
-        "data/mrp_lock_lot_data.xml",
-        "wizard/stock_transfer_details_view.xml",
         "wizard/mrp_product_produce_view.xml",
-        "wizard/wiz_lock_lot_view.xml",
-        "views/product_category_view.xml",
-        "views/stock_production_lot_view.xml",
-        "views/stock_quant_view.xml",
     ],
+    "auto": True,
```

Comment:
Don't you think is should?  all it contains now is the mrp-specific view to prevent adding blocked lots.